### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by abdulbasitkay**
update database information on README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ npm run start
 ```
 
 ## Database
-Assuming you already have user/role `quran_dev`, just run
+This project uses a Postgres database. Assuming you have Postgres installed create a user/role `quran_dev` then run the commands belo
```
Comment (sabeurthabti) [inline README.md]: `below`

**Comment by sabeurthabti** [comment]
needs to be updated to be `below` thanks

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:33014

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:33015

File: README.md (hunk 1 above)
Comment (abdulbasitkay) [reply README.md]: fixed

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:33018

**Pull request merged**